section1/exercises/loops.rb

```diff
@@ -5,18 +5,22 @@
 
 # Example: Write code that prints your name five times:
 5.times do
-  p "Hermione Granger"
+  puts "Matt Holmes"
 end
 
 # Write code that prints the sum of 2 plus 2 seven times:
 7.times do
-  # YOUR CODE HERE
+  puts 2 + 2
 end
 
 # Write code that prints the phrase 'She sells seashells down by the seashore'
 # ten times:
-# YOUR CODE HERE
+# 10.times do
+  puts "She sells seashells down by the seashore"
+end
 
 
 # Write code that prints the result of 5 + 7 a total of 9 timees
-# YOUR CODE HERE
+# 9.times do
+  puts 5 + 7
+end
```

Comment:
Uh oh - if you run this file there is a code break! In fact.. there are 2. Can you spot them and update?